frontend/components/card/CardTopicSelection.vue

```diff
@@ -9,18 +9,22 @@
       id="query"
       :display-value="() => query"
       :placeholder="$t('components.card-topic-selection.selector-placeholder')"
-      class="w-full py-2 pl-4 rounded-md text-light-special-text dark:text-dark-special-text bg-light-header dark:bg-dark-header elem-shadow-sm focus-brand"
+      class="topicInput w-full py-2 pl-4 rounded-md text-light-special-text dark:text-dark-special-text bg-light-header dark:bg-dark-header elem-shadow-sm focus-brand"
+      @keydown.tab.exact.prevent="inputTabbing($event)"
     />
     <ul class="hidden gap-2 sm:flex sm:flex-wrap">
       <ShieldTopic
-        v-for="t of filteredTopics"
+        v-for="(t, index) of filteredTopics"
         @click="selectTopic(t)"
         @keydown.enter.prevent="selectTopic(t)"
         :key="t.value"
         :topic="t.label"
-        class="max-sm:w-full"
+        class="topic max-sm:w-full"
         :active="isActiveTopic(t.value)"
         :isSelector="true"
+        @keydown.tab.prevent="topicTabbing($event)"
```

Comment:
Would you be able to fix the errors for attribute errors, @UnknownSean8, or do you want me to work on this? :)

Reply:
It's okay, I can fix it over the evening. Thanks! :D

Reply:
Thanks @UnknownSean8!